app/Customer.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Customer extends Model
+{
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array
+     */
+    protected $fillable = [
+        'email',
+        'password',
+        'company_id',
+        'firstname',
+        'lastname',
+        'phone',
+        'birthday',
+        'avatar',
+        'address',
+    ];
+
+    /**
+     * The attributes that should be hidden for arrays.
+     *
+     * @var array
+     */
+    protected $hidden = [
+        'password',
+    ];
+
+    /**
+     * Get the company that customer work for
+     */
+    public function company()
+    {
+        return $this->belongsTo('App\Company');
```

Comment:
`return $this->belongsTo('Company::classs');` c à.

Reply:
với cả phải đặt tất cả model trong folder `Model` cho dễ quản lý chứ

Reply:
@hapo-thomnt @hapo-thangbh `Models`